### ISSUE TITLE
Handling of custom chips

### ISSUE THREAD

**Issue by SportyScientist** [ncherric/Iliad#3]
My project ran on custom GSA Arrays, so it requires clustering. Is this something ILIAD can handle? :)

**Comment by ncherric** [comment]
Yes, Iliad can handle other arrays, but you will need to specify the GSA array files (e.g. specific product files and support files) in the configuration file where necessary. In my own work, I have made these same adjustments to the array file variables in config.yaml to process array data other than the MEGA array example provided.
